src/androidTest/java/database/GeneralInfoDaoTest.java

```diff
@@ -0,0 +1,102 @@
+package database;
+
+import android.content.Context;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.split.android.client.storage.db.GeneralInfoEntity;
+import io.split.android.client.storage.db.SplitRoomDatabase;
+
+public class GeneralInfoDaoTest {
```

Comment:
why there is no `remove` operation? are we sure that all the properties should exist always and we won't need to remove anything?

Reply:
if we add remove, please check making both operations, first update then delete, first delete then update. what would happen if you execute a delete and the data is not in the table, etc

Reply:
@mmelograno , thanks for the review!. For now, this table will be used only to store splits change number. That's why a remove method was not included.

Reply:
Could you please add it? Eventually we will add something.
cc: Thoughts @mredolatti ?

Reply:
Addressed in DM